airbyte-integrations/connectors/source-aircall/source_aircall/manifest.yaml

```diff
@@ -0,0 +1,214 @@
+version: "0.29.0"
+
+definitions:
+  requester:
+    type: HttpRequester
+    url_base: "https://api.aircall.io/v1"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_id'] }}"
+      password: "{{ config['api_token'] }}"
+
+  calls_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["calls"]
+      paginator:
+        type: NoPagination
+      requester:
+        $ref: "#/definitions/requester"
+    name: "calls"
+    $parameters:
+      path: "/calls"
+
+  company_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["company"]
```

Comment:
Please create a base RecordSelector and use `field_path: ["{{ parameters['name'] }}"]`